nautobot/docs/configuration/index.md

```diff
@@ -6,11 +6,11 @@ This section describes how to get started with configuring Nautobot.
 
 An initial configuration can be created by executing `nautobot-server init`. This will generate a new configuration with all of the default settings provided for you, and will also generate a unique [`SECRET_KEY`](required-settings.md#secret_key).
 
-By default the file will be created at `$HOME/.nautobot/nautobot_config.py`:
+By default the file will be created at `$NAUTOBOT_ROOT/nautobot_config.py`:
```

Comment:
Note that if `$NAUTOBOT_ROOT` isn't set, it does in fact default to `~/.nautobot`. This document doesn't cover the `$NAUTOBOT_ROOT` variable until a couple of sections later below. Perhaps instead:

```suggestion
By default (if you haven't set [`NAUTOBOT_ROOT`](#nautobot-root-directory) to some other value), the file will be created at `$HOME/.nautobot/nautobot_config.py`:
```